includes/Helpers/Plugin.php

```diff
@@ -108,6 +108,10 @@ protected function get_admin_users(): array {
 
 		// Add administrators to the $users and check for super admin
 		foreach ( $admin_users as $user ) {
+			if ( empty( $user->user_email ) ) {
+				continue;
+			}
+
 			$users[] = array(
 				'id'    => $user->ID,
 				'email' => $user->user_email,
```

Comment:
@desrosj Any idea why `$user->user_email` could be `null` here?

<img width="529" alt="Screenshot 2025-05-30 at 9 33 25 AM" src="https://github.com/user-attachments/assets/f00b47ad-749e-40ca-8158-07b53f8d1993" />

Reply:
I found the culprit: [ConvertEmptyStringsToNull middleware](https://github.com/laravel/framework/blob/11.x/src/Illuminate/Foundation/Http/Middleware/ConvertEmptyStringsToNull.php)